docs/upgrades/v0.14-to-v1-upgrade/upgraded-examples/src/test/kotlin/examples/ErrorHandlingChanges.kt

```diff
@@ -0,0 +1,164 @@
+package examples
+
+import org.junit.jupiter.api.assertThrows
+import org.partiql.eval.Mode
+import org.partiql.eval.compiler.PartiQLCompiler
+import org.partiql.parser.PartiQLParser
+import org.partiql.planner.PartiQLPlanner
+import org.partiql.spi.Context
+import org.partiql.spi.SourceLocation
+import org.partiql.spi.catalog.Session
+import org.partiql.spi.errors.PError
+import org.partiql.spi.errors.PErrorKind
+import org.partiql.spi.errors.PErrorListener
+import org.partiql.spi.errors.PRuntimeException
+import org.partiql.spi.errors.Severity
+import org.partiql.spi.types.PType
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class ErrorHandlingChanges {
+
+    /**
+     * The example shows the default behavior of error handling in major PartiQL components {@link PartiQLParser},
+     * {@link PartiQLPlanner} and {@link PartiQLCompiler}. {@link PRuntimeException} will be thrown from the PartiQL system
+     * to trap any parsing errors or warnings. {@link PRuntimeException} wraps a {@link PError} which includes {@link PError#code()},
+     * {@link PError#kind}, {@link PError#severity}, {@link PError#location} and {@link PError#properties}.
+     * {@link PRuntimeException} with some enhanced information allows us for more robust and customized error control.
+     */
+    @Test
+    fun `Default PartiQL error handling`() {
```

Comment:
For these examples, let's try to keep the naming and formatting consistent. In the v0.14.9 examples, you started off with showing a parser example, planner example, (and should also include an evaluation example). It's hard to follow the differences between versions if we change the naming and content of the examples. We can leave the additional example related to custom error handlers as an extra example in v1.